graylog2-server/src/main/java/org/graylog2/grok/InMemoryGrokPatternService.java

```diff
@@ -34,6 +34,8 @@
 import java.util.regex.PatternSyntaxException;
 import java.util.stream.Collectors;
 
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
Minor nitpick: This *could* be replaced by [`Objects#requireNonNull()`](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#requireNonNull-T-).